src/main/java/net/dancier/resources/RecommendationsResource.java

```diff
@@ -0,0 +1,29 @@
+package net.dancier.resources;
+
+import net.dancier.service.RecommendationsService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@Path("/recommendations")
+public class RecommendationsResource {
+
+    public static Logger logger = LoggerFactory.getLogger(RecommendationsResource.class);
+
+    private RecommendationsService recommendationsService;
+
+    public RecommendationsResource(RecommendationsService recommendationsService) {
+        this.recommendationsService = recommendationsService;
+    }
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getIt() {
```

Comment:
One could argue if this is a good name ;) I'd suggest something like readRecommendation or getRecommandation or getSingleRecord or something.

Reply:
would you be happy with: readRecommandations?

Reply:
Absolutely